### PR TITLE
Longhorn

### PR DESCRIPTION
**Pull request by imikushin**

### CODE REVIEW

File: infra-templates/longhorn/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: '2'
+services:
+  longhorn-driver:
+    privileged: true
+    pid: host
+    image: imikushin/storage-longhorn:dev45
```
Comment (deniseschannon) [inline infra-templates/longhorn/0/docker-compose.yml]: can we use a rancher image version?

File: infra-templates/longhorn/0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    environment:
+      RANCHER_DEBUG: ${RANCHER_DEBUG}
+    logging:
+      driver: json-file
+      options:
+        max-size: 25m
+        max-file: '2'
+
+  longhorn-orc:
+    image: imikushin/longhorn-orc:8d72167
```
Comment (deniseschannon) [inline infra-templates/longhorn/0/docker-compose.yml]: same, can we have a rancher version?

File: infra-templates/longhorn/0/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    volumes:
+    - /var/run:/var/run
+    volumes_from:
+    - longhorn-binary   # /usr/local/bin
+    environment:
+      RANCHER_DEBUG: ${RANCHER_DEBUG}
+      LONGHORN_IMAGE: imikushin/longhorn:bca578f
+
+  longhorn-binary:
+    image: imikushin/longhorn:bca578f
```
Comment (deniseschannon) [inline infra-templates/longhorn/0/docker-compose.yml]: same thing, rancher version

File: infra-templates/longhorn/0/docker-compose.yml (hunk 1 above)
Comment (imikushin) [reply infra-templates/longhorn/0/docker-compose.yml]: There's no rancher image version for it, yet. The PR is in there: rancher/storage#27

File: infra-templates/longhorn/0/docker-compose.yml (hunk 2 above)
Comment (imikushin) [reply infra-templates/longhorn/0/docker-compose.yml]: The repo hasn't been migrated to rancher yet.

File: infra-templates/longhorn/0/docker-compose.yml (hunk 3 above)
Comment (imikushin) [reply infra-templates/longhorn/0/docker-compose.yml]: @yasker Can you build a new version from latest longhorn master?

File: infra-templates/longhorn/0/rancher-compose.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.catalog:
+  name: Longhorn
+  version: v0.4.0
```
Comment (deniseschannon) [inline infra-templates/longhorn/0/rancher-compose.yml]: can we start at v0.1.0

File: infra-templates/longhorn/config.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+name: Longhorn
+description: |
+  Block storage built for, by, and on Rancher
+version: v0.4.0
```
Comment (deniseschannon) [inline infra-templates/longhorn/config.yml]: can we match the v0.1.0 version in rancher-compose.yml

File: infra-templates/longhorn/0/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    environment:
+      RANCHER_DEBUG: ${RANCHER_DEBUG}
+    logging:
+      driver: json-file
+      options:
+        max-size: 25m
+        max-file: '2'
+
+  longhorn-orc:
+    image: imikushin/longhorn-orc:55230a5
```
Comment (deniseschannon) [inline infra-templates/longhorn/0/docker-compose.yml]: same thing about rancher version

**Comment by deniseschannon** [comment]
overall, we should make sure to use real rancher versions before we start testing.

File: infra-templates/longhorn/0/docker-compose.yml (hunk 3 above)
Comment (yasker) [reply infra-templates/longhorn/0/docker-compose.yml]: @imikushin `rancher/longhorn:bca578f` is available now as latest Longhorn image.

**Comment by imikushin** [comment]
@deniseschannon fixed

**Comment by imikushin** [comment]
set the version to v0.1.0

**Pull request merged**